calico-enterprise/operations/troubleshoot/commands.mdx

```diff
@@ -237,8 +237,8 @@ items:
 - apiVersion: operator.tigera.io/v1
   kind: Installation
   metadata:
-    - apiVersion: operator.tigera.io/v1
- spec:
+    name: default
```

Comment:
Why this change?

Reply:
Reverted the change. Fixed by adjusting the indentation.